docs/index.html

```diff
@@ -206,6 +206,28 @@ <h2>
           <span>Experiments</span>
         </h2>
 
+        <details id="warc-gpt">
+          <summary role="button" class="contrast">
+            <span role="heading" aria-level="3">WARC-GPT</span>
+          </summary>
+
+          <h4>Because we wondered:</h4>
+
+          <p><strong>Can the techniques used to ground and augment the responses provided by Large Language Models be used to help explore web archive collections?</strong></p>
+
+          <p>This is the question we’ve asked ourselves as part of our ongoing explorations of how artificial intelligence changes our relationship to knowledge, which led us to develop and release WARC-GPT: an experimental open-source Retrieval Augmented Generation tool for exploring collections of WARC files using AI.</p>
```

Comment:
```suggestion
          <p>This is the question we’ve asked ourselves while exploring how artificial intelligence changes our relationship to knowledge, which led us to develop and release WARC-GPT: an experimental open-source retrieval-augmented generation tool for exploring collections of WARC files using AI.</p>
```

Reply:
You'd think I would have remembered to add an hyphen from from the other PR I submitted 20 minutes ago ... nope 😆